config/data/langs.toml

```diff
@@ -368,7 +368,7 @@ console.log 'Hello, World!'
 
 // Looping
 for i of [0..9]
-    console.log i
+    console.log `${i}`
```

Comment:
This seems like an unrelated change, or if it isn't unrelated are we saying that adding FORCE_COLOR affects stdout and therefore existing programs? If so I don't think this'll fly.

Reply:
This wasn't supposed to still be open, and thought it was a draft already.

> This seems like an unrelated change, or if it isn't unrelated are we saying that adding FORCE_COLOR affects stdout and therefore existing programs? If so I don't think this'll fly.

Yes, you're right. Odd stuff, for now. I know that's out of the question.